plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/exporter/DataExporterCSV.java

```diff
@@ -249,6 +257,34 @@ public void exportRow(DBCSession session, DBCResultSet resultSet, Object[] row)
         writeRowLimit();
     }
 
+    private String editArrayPrefixAndSuffix(DataExporterArrayFormat modifiedFormat,String stringValue) {
```

Comment:
I'm unsure how I feel about manually replacing all characters in an already formatted string. But we don't have any other way to nicely approach that, unfortunately...

Reply:
One alternative approach I can think of is:

Make a singleton class in a common module and if the datatype is Array, add a new DBDisplayFormat called as EXPORT, set the brackets and use it in the plugins, if the format is EXPORT.

Not sure of how dbeaver would manage it's state, and this would also make plugins to undergo changes.

Reply:
I agree. I think the approach taken is good enough for that case.